lib/citizens_advice_form_builder/elements/collections/radio_buttons.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module CitizensAdviceFormBuilder
+  module Elements
+    module Collections
+      class RadioButtons < Base
+        include ActionView::Helpers::FormOptionsHelper
+
+        def render
+          component = CitizensAdviceComponents::RadioGroup.new(
+            legend: label,
+            name: field_id,
+            options: {
```

Comment:
Would it be possible to add `layout` option here? There are a few places in the designs where the radio buttons are inlined. If it's difficult to add as part of this PR, it's non blocking as I think we can live without inline styles for those for now.

Reply:
It's a small enough change, but I think I'll merge this as-is and add a ticket to add this separately.